reco_utils/recommender/geoimc/geoimc_utils.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+from sklearn.decomposition import PCA
+
+from reco_utils.dataset.download_utils import maybe_download
+from IPython import embed
+
+def length_normalize(matrix):
+    """Length normalize the matrix
+
+    Args:
+        matrix (np.ndarray): Input matrix that needs to be normalized
+
+    Returns:
+        Normalized matrix
+    """
+    norms = np.sqrt(np.sum(matrix**2, axis=1))
+    norms[norms == 0] = 1
+    return matrix / norms[:, np.newaxis]
+
+
+def mean_center(matrix):
+    """Performs mean centering across axis 0
+
+    Args:
+        matrix (np.ndarray): Input matrix that needs to be mean centered
+    """
+    avg = np.mean(matrix, axis=0)
+    matrix -= avg
+
+
+def binarize(a, threshold):
```

Comment:
in relation to [this comment](https://github.com/microsoft/recommenders/pull/1142#discussion_r454849136), maybe a good place to put this function would be https://github.com/microsoft/recommenders/blob/staging/reco_utils/common/python_utils.py

Reply:
Resolved at 6a3a5aa0bbfe79536c254bed4a7e77ecbb20c1d3